server/main.lua

```diff
@@ -96,7 +96,7 @@ lib.callback.register('garbagejob:server:EndShift', function(source)
 end)
 
 lib.callback.register('garbagejob:server:spawnVehicle', function(source, coords)
-    local netId = QBCore.Functions.CreateVehicle(source, joaat(Config.Vehicle), coords, false)
+    local netId = CreateVehicle(source, joaat(Config.Vehicle), coords, false)
```

Comment:
This should actually be [SpawnVehicle](https://qbox-project.github.io/resources/core/functions/server#spawnvehicle)